### PR TITLE
Separate the testcases

### PR DESCRIPTION
**Pull request by phamthai1912**

### CODE REVIEW

File: src/test/java/com/client/tests/Board/TC002_GetBoard.java [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.Matchers.is;
+
+public class TC002_GetBoard extends BaseTest {
+    private String boardName = FakerUtils.generateName();
+    private String boardID;
+
+    @BeforeTest
```
Comment (dinhbao10t4) [inline src/test/java/com/client/tests/Board/TC002_GetBoard.java:20]: It should be @BeforeMethod

File: src/test/java/com/client/tests/Board/TC002_GetBoard.java [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    public void prepareData() {
+        Board board = BoardApi.createBoard(boardName).as(Board.class);
+        boardID = board.getId();
+    }
+
+    @AfterTest
```
Comment (dinhbao10t4) [inline src/test/java/com/client/tests/Board/TC002_GetBoard.java:26]: It should be @AfterMethod

File: src/test/java/com/client/tests/Board/TC003_DeleteBoard.java [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+    private String boardID;
+
+    @BeforeTest
+    public void prepareData() {
+        Board board = BoardApi.createBoard(boardName).as(Board.class);
+        boardID = board.getId();
+    }
+
+    @Test(description = "TC003 - Delete a board")
+    public void getCreatedBoard() {
```
Comment (dinhbao10t4) [inline src/test/java/com/client/tests/Board/TC003_DeleteBoard.java:25]: the method name should be deleteBoard

File: src/test/java/com/client/tests/Board/TC003_DeleteBoard.java [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+import org.testng.annotations.Test;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.is;
+
+public class TC003_DeleteBoard extends BaseTest {
+    private String boardName = FakerUtils.generateName();
+    private String boardID;
+
+    @BeforeTest
```
Comment (dinhbao10t4) [inline src/test/java/com/client/tests/Board/TC003_DeleteBoard.java:18]: same as above

File: src/test/java/com/client/tests/Board/TC003_DeleteBoard.java (hunk 3 above)
Comment (phamthai1912) [reply src/test/java/com/client/tests/Board/TC003_DeleteBoard.java]: Updated

File: src/test/java/com/client/tests/Board/TC002_GetBoard.java (hunk 1 above)
Comment (phamthai1912) [reply src/test/java/com/client/tests/Board/TC002_GetBoard.java:20]: 1 class only contain 1 testcase, hence @BeforeMethod ~ @BeforeTest. Using BeforeTest in this situation will help more clear meaning.

**Review by dinhbao10t4 — APPROVED** [review]

**Pull request merged**